README.md

```diff
@@ -59,20 +59,27 @@ For more details see: [Multi Cluster on isolated networks](#multi-cluster-on-iso
 
 ### Version mapping
 Implementation of additional features brings complexity and sometimes may happen that extra feature is broken in special cases. 
-To prevent potential issues due to usage such versions, the k3d, k3s versions are hard-coded.
+To prevent potential issues due to usage such versions, the k3d version is hard-coded.
```

Comment:
So with this change it is not hard-coded anymore, rather it is predefined?

Reply:
thx a lot an advice, amending